src/Services/Tesseract.php

```diff
@@ -8,15 +8,21 @@ class Tesseract extends OcrAbstract
 
     protected $command;
 
-    public function scan($imagePath) {
+    public function scan($imagePath, $lang = null) {
 
+        if ($lang === null) {
+            $lang = env('OCR_LANG');
+        }
         $this->setImagePath($imagePath);
         $shell = new Shell();
 
         $executable = config('lara_ocr.engines.tesseract.executable', 'tesseract');
 
-        $command = trim("$executable $imagePath stdout quiet");
+        $command = trim($executable.(($lang !== null) ? ' -l '.$lang : '').' '.$imagePath.' stdout quiet');
```

Comment:
@Henning256 
Thanks a lot for the PR. 

Can you put this tarnery operator in a different line? Like:

```
$langOption = is_null($lang) ? "" : " -l {$lang}";
$command = trim("$executable $langOption $imagePath stdout quiet");
```

I believe it will be more readable in this way.